unblob/handlers/__init__.py

```diff
@@ -2,7 +2,7 @@
 from .archive import ar, arc, arj, cab, cpio, dmg, rar, sevenzip, stuffit, tar, zip
```

Comment:
add reference to the specification in the commit

Reply:
There's no open specs. Some online content that talks about it:
- https://foxglovesecurity.com/2017/11/20/a-sheep-in-wolfs-clothing-finding-rce-in-hps-printer-fleet/ (hardly understood the whole format)
- https://labs.withsecure.com/content/dam/labs/docs/Printing-Shellz.pdf (first reversing)
- https://github.com/tylerwhall/hpbdl (extractor in Rust)